util/travis/script.sh

```diff
@@ -3,21 +3,61 @@
 
 needs_compile || exit 0
 
+function perform_lint() {
+	CLANG_FORMAT=clang-format-3.9
+	${CLANG_FORMAT} --version
+
+	if [ "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then
+		# Get list of every file modified in this pull request
+		files_to_lint="$(git diff --name-only --diff-filter=ACMRTUXB $TRAVIS_COMMIT_RANGE | grep '^src/[^.]*[.]\(cpp\|h\)$' || true)"
+	else
+		# Check everything for branch pushes
+		files_to_lint="$(find src/ -name '*.cpp' -or -name '*.h' | egrep -v "src/(gmp|lua|jsoncpp)")"
+	fi
+
+	for f in ${files_to_lint}; do
+		d=$(diff -u "$f" <(${CLANG_FORMAT} "$f") || true)
+		if ! [ -z "$d" ]; then
+			echo "$f not compliant to coding style, here is the fix:"
+			echo "$d"
```

Comment:
I suggest replacing with:
`printf "The file %s is no compliant with the coding style:\n%s\n" "$f" "$d"`